### PR TITLE
Enable the PVH header in kernel config in order to use cloud-hypervisor and direct kernel boot

### PR DESCRIPTION
**Pull request by dingelish**
For TDX enablement, we may want to use cloud hypervisor, TDShim and direct kernel boot. This stack [requires](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/1db30405e13995172bd45386a49a54e7c7a5f621/README.md?plain=1#L111) us to build the guest kernel with [PVH header](https://wiki.xenproject.org/wiki/Understanding_the_Virtualization_Spectrum#PVH).

I manually tested this kernel with CH+Oak guest image and it works.

### CODE REVIEW

**Review by tiziano88 — APPROVED** [review]
LG, please wait for another approval too

**Review by conradgrobler — APPROVED** [review]

**Pull request merged**
